### DISCUSSION TITLE
MS SQL Support

### DISCUSSION THREAD

**Discussion by jesseg34** [directus/directus#4799] [Feature Requests]
I just starting evaluating directus for a new project and have been very impressed. I would like to use MS SQL for storage however I see that its still in Alpha. I'm wondering where the community is at with completing that feature as I didn't see any issues for it. Thanks!

**Comment by rijkvanzanten** [comment]
Heya! It should be good to go! 🙂 There were a couple issues on the MS SQL side before, so we left it marked Alpha. That being said, out [those issues](https://github.com/directus/directus/issues/4644), there's only 1 unresolved issue left: https://github.com/directus/directus/issues/4644.

As a matter of fact, I think it's time to drop the Alpha label (cc @benhaynes )

**Reply by jesseg34** [reply]
That is great news! I'll give it another shot later this afternoon. Initially I tried using MSSQL but my instance could not connect to SQL Express but that's probably user error :) I switched to PostgreSQL to test out the features.

**Reply by benhaynes** [reply]
Yes, let's go ahead and update the CLI to reflect the status of our SQL vendors! Do you need a ticket for that @rijkvanzanten ?
